scripts/runClangFormat.sh

```diff
@@ -1,16 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/sh
 #
 # Copyright 2020-2022 The Mumble Developers. All rights reserved.
 # Use of this source code is governed by a BSD-style license
 # that can be found in the LICENSE file at the root of the
 # Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
-currentDir=$(dirname $0)
+set -o pipefail
```

Comment:
I get an illegal option error on this line

Reply:
> I get an illegal option error on this line

Which system/shell?
I just removed it.  I know not **all** shells have it, but I hoped for all relevant ones to have it.

Reply:
Ha - good question. I always use bash so I don't even know what my regular shell is ^^
I'll go check

Reply:
So apparently this one is called "dash" :shrug: